GETTING_STARTED.md

```diff
@@ -89,6 +89,10 @@ If necessary, you can update some settings for the extension.
 
 1. Keyboard shortcuts can be changed by searching for `qiskit-vscode` in the Keyboard Shortcuts settings
 
+1. The Telemetry can be disabled by unchecking the `qiskitCodeAssistant.enableTelemetry` setting or turning `off` the `telemetry.telemetryLevel` setting:
+
+   > **NOTE**: The telemetry does not collect your code nor the suggested code completions. What is collected is whether a code suggestion was accepted or dismissed.
+
```

Comment:
```suggestion
2. The Telemetry can be disabled by unchecking the `qiskitCodeAssistant.enableTelemetry` setting or turning `off` the `telemetry.telemetryLevel` setting:
```
shouldn't it be `2` in the numbering?

Reply:
Let's add the same to the README after the https://github.com/Qiskit/qiskit-code-assistant-vscode?tab=readme-ov-file#keyboard-shortcuts so we do the same as in jupyterlab

Reply:
> shouldn't it be `2` in the numbering?

it doesnt matter. the markdown preview will update/display the numbering correctly. i tend to just use all `1.`'s so i dont have to worry about keeping them updated accordingly as we add/remove items from list. 

> Let's add the same to the README after the https://github.com/Qiskit/qiskit-code-assistant-vscode?tab=readme-ov-file#keyboard-shortcuts so we do the same as in jupyterlab

done!